sdk/spring/spring-cloud-azure-test-appconfiguration-config/pom.xml

```diff
@@ -13,8 +13,8 @@
   </parent>
 
     <groupId>com.azure.spring</groupId>
-    <artifactId>azure-spring-cloud-test-appconfiguration-config</artifactId>
-    <version>1.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-test-appconfiguration-config;current} -->
+    <artifactId>spring-cloud-azure-test-appconfiguration-config</artifactId>
```

Comment:
How about naming this `spring-cloud-azure-integration-test-appconfiguration-config`

Reply:
Have we added this to the tests.yml?